pages/common/host.md

```diff
@@ -13,3 +13,7 @@
 - Reverse lookup an IP:
 
 `host {{ip_address}}`
+
+- Specify an alternate DNS server to query (8.8.8.8 is Google's public DNS):
```

Comment:
I wonder if it is necessary to specify the "(8.8.8.8 is ..)". Even if people do not know what 8.8.8.8, the description is clear that 8.8.8.8 is a specific dns server which is used to query.

Reply:
I have copied this description from dig https://github.com/tldr-pages/tldr/blob/master/pages/common/dig.md

If it does not necessary - maybe remove it from there too?

Reply:
Yes, I think so.

Reply:
I created the separate PR https://github.com/tldr-pages/tldr/pull/2447, cause [contributing guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request)